owncloudApp/build.gradle

```diff
@@ -26,6 +26,7 @@ dependencies {
 
     // Image loading
     implementation 'com.github.bumptech.glide:glide:4.12.0'
+    implementation 'com.github.corouteam:GlideToVectorYou:v2.0.0'
```

Comment:
Is this dependency needed?

Reply:
Why do you repeat the comments? Is it a bug? haha 😆